### PR TITLE
Add new strings

### PR DESCRIPTION
**Pull request by peiying2**
- new strings in newsletter/developer.html per bug 1331504
- new strings in home.html per bedrock PR #4591

### CODE REVIEW

**Comment by flodolo** [comment]
What about https://github.com/mozilla/bedrock/pull/4587

Always remember to check **all** PRs tagged l10n, including the one closed.

**Comment by flodolo** [comment]
Side question: who's supposed to translated the new IHR page? I don't think it makes sense to rush these out if we need to expose that page as well.

**Comment by flodolo** [comment]
Newsletter strings look good in the meantime, file is correct
https://github.com/mozilla/bedrock/blob/master/bedrock/newsletter/templates/newsletter/developer.html#L7

**Comment by TheoChevalier** [comment]
> Side question: who's supposed to translated the new IHR page? I don't think it makes sense to rush these out if we need to expose that page as well.

If you’re talking about https://www.mozilla.org/fr/internet-health/ it’s localized, so that would be great to have the link added today since the report is live. The subpage for Security and Privacy that will go live for Privacy day should be available today, but I’m not sure there’s a bedrock PR yet.

**Comment by flodolo** [comment]
> The subpage for Security and Privacy that will go live for Privacy day should be available today, but I’m not sure there’s a bedrock PR yet.

That's what I was talking about, the PR was merged this morning.

File: en-US/mozorg/internet-health.lang [hunk 1]
```diff
@@ -108,6 +99,14 @@ Get started with the Mozilla Learning Network
 Stay informed
 
 
+;Check out Mozilla’s <br>Internet Health Report
```
Comment (flodolo) [inline en-US/mozorg/internet-health.lang]: Where' the tag for these strings?

File: en-US/mozorg/internet-health/privacy-security.lang [hunk 2]
```diff
@@ -0,0 +1,188 @@
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/privacy-security.html
```
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: This URL doesn't work

File: en-US/mozorg/newsletters.lang [hunk 3]
```diff
@@ -44,6 +44,18 @@ Mozilla touches on a variety of important issues.
...
 
+;Love the web? So do we!
+Love the web? So do we!
+
+
+;Join thousands of developers like you who are learning the best of web development.
+Join thousands of developers like you who are learning the best of web development.
+
+
+;;If you don’t like the Mozilla developer newsletter, you can easily unsubscribe (but we think you’ll like it).
```
Comment (flodolo) [inline en-US/mozorg/newsletters.lang]: What happened here? You added an extra ;

File: en-US/mozorg/internet-health/privacy-security.lang (hunk 2 above)
Comment (flodolo) [inline en-US/mozorg/internet-health/privacy-security.lang]: It should be https://www-dev.allizom.org/it/internet-health/privacy-security/ (also fix following line)

File: en-US/mozorg/internet-health.lang (hunk 1 above)
Comment (peiying2) [inline en-US/mozorg/internet-health.lang]: Added to both strings.

File: en-US/mozorg/newsletters.lang (hunk 3 above)
Comment (peiying2) [inline en-US/mozorg/newsletters.lang]: arg.. fixed.

**Pull request merged**